apps/src/templates/rubrics/AiAssessmentFeedback.jsx

```diff
@@ -12,12 +13,12 @@ import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import Checkbox from '@cdo/apps/componentLibrary/checkbox/Checkbox';
 import Button from '@cdo/apps/templates/Button';
 
-export default function AiAssessmentFeedback({learningGoalKey}) {
-  const radioGroupName = `ai-assessment-feedback-${learningGoalKey}`;
-  const thumbsupval = 'thumbsup';
-  const thumbsdownval = 'thumbsdown';
+export default function AiAssessmentFeedback({aiInfo}) {
+  const radioGroupName = `ai-assessment-feedback-${aiInfo.id}`;
```

Comment:
this object appears to be of type aiEvaluationShape, but I don't see `id` defined on that shape. can you please add it here? https://github.com/code-dot-org/code-dot-org/blob/6c6897ec21ee803689035f609983c633beb73b5f/apps/src/templates/rubrics/rubricShapes.jsx#L45-L49

Reply:
That's already in this PR!

Reply:
apologies! 😊